src/dotnet/commands/dotnet-test3/Program.cs

```diff
@@ -0,0 +1,233 @@
+﻿// Copyright(c) .NET Foundation and contributors.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.MSBuild;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Tools.Test3
+{
+    public class Test3Command
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            var cmd = new CommandLineApplication(throwOnUnexpectedArg: false)
+            {
+                Name = "dotnet test3",
+                FullName = ".NET Test Driver",
+                Description = "Test Driver for the .NET Platform"
+            };
+
+            cmd.HelpOption("-h|--help");
+
+            var argRoot = cmd.Argument(
+                "<PROJECT>",
+                "The project to test, defaults to the current directory.",
+                multipleValues: false);
+
+            var settingOption = cmd.Option(
+                "--settings <SettingsFile>",
+                "Settings to use when running tests." + Environment.NewLine,
+                CommandOptionType.SingleValue);
+
+            var listTestsOption = cmd.Option(
+                "-lt|--listTests",
```

Comment:
We are using the same name for argument which we use in vstest.console to give similar experience everywhere.